modules/merge_waiter/main.tf

```diff
@@ -1,3 +1,10 @@
+# prepare lambda zip file
```

Comment:
this code already exists in mater.

Reply:
if it appears in master how come it appears as a change ?